rir/src/analysis/Printer.h

```diff
@@ -1,22 +1,21 @@
 #pragma once
```

Comment:
I would suggest a `tools` or `utils` folder for the things like the printer, because it is neither an optimization under which we filed it before.